pkg/webhook/sidecar_spec.go

```diff
@@ -118,6 +113,101 @@ func GetSidecarContainerSpec(c *Config) corev1.Container {
 	return container
 }
 
+// GetSecurityContext ensures the sidecar that uses it follows Restricted Pod Security Standard.
+// See https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
+func GetSecurityContext() *corev1.SecurityContext {
+	return &corev1.SecurityContext{
+		AllowPrivilegeEscalation: ptr.To(false),
+		ReadOnlyRootFilesystem:   ptr.To(true),
+		Capabilities: &corev1.Capabilities{
+			Drop: []corev1.Capability{
+				corev1.Capability("ALL"),
+			},
+		},
+		SeccompProfile: &corev1.SeccompProfile{Type: corev1.SeccompProfileTypeRuntimeDefault},
+		RunAsNonRoot:   ptr.To(true),
+		RunAsUser:      ptr.To(int64(NobodyUID)),
+		RunAsGroup:     ptr.To(int64(NobodyGID)),
+	}
+}
+
+func (si *SidecarInjector) GetNativeMetadataPrefetchSidecarContainerSpec(pod *corev1.Pod, image string) corev1.Container {
+	container := si.GetMetadataPrefetchSidecarContainerSpec(pod, image)
+	container.Env = append(container.Env, corev1.EnvVar{Name: "NATIVE_SIDECAR", Value: "TRUE"})
+	container.RestartPolicy = ptr.To(corev1.ContainerRestartPolicyAlways)
+
+	return container
+}
+
+func getMetadataPrefetchConfig(image string) *Config {
+	return &Config{
+		CPURequest:              resource.MustParse("10m"),
```

Comment:
please create constants for these e.g metadataSidecarCPURequest

Reply:
Done